source/_integrations/iammeter.markdown

```diff
@@ -13,12 +13,13 @@ ha_platforms:
   - sensor
 ---
 
-`iammeter` provides real-time readings of single-phase (WEM3080, WEM3162) and three-phase (WEM3080T) meters from [IAMMETER](https://www.iammeter.com) over Wi-Fi.
+`IAMMETER` provides real-time readings of single-phase (WEM3080) and three-phase (WEM3080T) meters from [IAMMETER](https://www.iammeter.com/docs/homeassistant) over Wi-Fi.
```

Comment:
```suggestion
IAMMETER provides real-time readings of single-phase (WEM3080) and three-phase (WEM3080T) meters from [IAMMETER](https://www.iammeter.com/docs/homeassistant) over Wi-Fi.
```

Reply:
Why does this link to docs and not a product?

Reply:
I think this link is like a step-by-step tutorial about how to integrate the IAMMETER products into HA.

Reply:
Yes, but the link doesn't suggest that.

Reply:
Hi, Frenck, 
Thanks for your suggestion.
I have replaced the tutorial page with the official web page of IAMMETER and added two products page.